Source/FakeItEasy/Creation/DefaultFakeCreatorFacade.cs

```diff
@@ -126,6 +126,11 @@ public IFakeOptionsBuilder<T> Implements(Type interfaceType)
                 return this;
             }
 
+            public IFakeOptionsBuilder<T> Implements<TInterface>()
+            {
+                return Implements(typeof (TInterface));

```

Comment:
I also use ReSharper ; probably a different configuration.

 

But I was not getting any compiler warning… the compiler doesn’t care one bit about whitespace. Are you sure the warning you were seeing was from the compiler?

 

From: Blair Conrad [mailto:notifications@github.com] 
Sent: dimanche 29 mars 2015 01:33
To: FakeItEasy/FakeItEasy
Cc: Thomas Levesque
Subject: Re: [FakeItEasy] Add generic overload of Implements (#471)

 

In Source/FakeItEasy/Creation/DefaultFakeCreatorFacade.cs <https://github.com/FakeItEasy/FakeItEasy/pull/471#discussion_r27350132> :

> @@ -126,6 +126,11 @@ public IFakeOptionsBuilder<T> Implements(Type interfaceType)
>                  return this;
>              }
>  
> +            public IFakeOptionsBuilder<T> Implements<TInterface>()
> +            {
> +                return Implements(typeof (TInterface));

You're very accommodating.

Yeah, for number 3, it was the space that was the problem. I'd lost my backticks when copying my comment and didn't notice.
Not sure about VS adding it. For me it doesn't, but that could be due to my VS configuration or the fact that I have ReSharper (and its configuration, I suppose). Dealing with various projects' conventions is a pain, no?
The warnings emitted by the compilation can be your guide—FakeItEasy usually builds without warnings.

—
Reply to this email directly or view it on GitHub <https://github.com/FakeItEasy/FakeItEasy/pull/471/files#r27350132> .  <https://github.com/notifications/beacon/ABxyWVR4lVrYebo74i9Fj86Tsm_ojlv_ks5n5z-jgaJpZM4D16jk.gif>